ecs-cli/modules/cli/compose/adapter/convert_test.go

```diff
@@ -548,3 +548,19 @@ func TestSortedGoString(t *testing.T) {
 
 	assert.Equal(t, strA, strB, "Sorted inputs should match")
 }
+
+func TestConvertCamelCaseToUnderScore(t *testing.T) {
```

Comment:
If you want to get points from @petderek, refactor this to be a [table driven test](https://github.com/golang/go/wiki/TableDrivenTests).

Reply:
Golang has a [subtest feature](https://blog.golang.org/subtests) that makes this awesome. The test output makes it really clear which case broke (this is stolen from the linked blog post):

```
--- FAIL: TestTime (0.00s)
    --- FAIL: TestTime/12:31_in_Europe/Zuri (0.00s)
        time_test.go:84: could not load location
    --- FAIL: TestTime/12:31_in_America/New_York (0.00s)
        time_test.go:88: got 07:31; want 7:31
```

Reply:
Awesome! Added! (also: anything for brownie and/or fake internet points)
```
=== RUN   TestConvertCamelCaseTo_UnderScore
=== RUN   TestConvertCamelCaseTo_UnderScore/CamelCaseString_should_be_camel_case_string
=== RUN   TestConvertCamelCaseTo_UnderScore/lowercase_should_be_lowercase
=== RUN   TestConvertCamelCaseTo_UnderScore/UPPERCASE_should_be_uppercase
=== RUN   TestConvertCamelCaseTo_UnderScore/CamelWithACRONYM_should_be_camel_with_acronym
=== RUN   TestConvertCamelCaseTo_UnderScore/Uppercase_should_be_uppercase
--- PASS: TestConvertCamelCaseTo_UnderScore (0.00s)
    --- PASS: TestConvertCamelCaseTo_UnderScore/CamelCaseString_should_be_camel_case_string (0.00s)
    --- PASS: TestConvertCamelCaseTo_UnderScore/lowercase_should_be_lowercase (0.00s)
    --- PASS: TestConvertCamelCaseTo_UnderScore/UPPERCASE_should_be_uppercase (0.00s)
    --- PASS: TestConvertCamelCaseTo_UnderScore/CamelWithACRONYM_should_be_camel_with_acronym (0.00s)
    --- PASS: TestConvertCamelCaseTo_UnderScore/Uppercase_should_be_uppercase (0.00s)
```